cypress/integration/mentions.ts

```diff
@@ -0,0 +1,28 @@
+describe('mentions example', () => {
+  before(() => cy.visit('examples/mentions'))
```

Comment:
Interesting, what's the difference between `before()` and `beforeEach()`?

Reply:
`before` runs one time at the beginning of the entire suite (the whole `mentions.ts`). `beforeEach` runs right before every `it` block.

For that reason, I believe `beforeEach` should be used here, because I would assume `visit`ing a page again between one test and another resets the state of the page - which should make the state of each test clean (and not depend on how the previous test left things)

```suggestion
  beforeEach(() => cy.visit('examples/mentions'))
```